src/main/scala/com/ambiata/disorder/PositiveLongSmall.scala

```diff
@@ -0,0 +1,16 @@
+package com.ambiata.disorder
+
+import org.scalacheck._
+import scalaz._, Scalaz._
+
+case class PositiveLongSmall(value: Long)
```

Comment:
> nice to provide the different combinations.

I might be tempted to just have `toLong` on `PositiveIntSmall`, but I'm not sure if that's better/worse than just calling `value.toLong`.